.husky/pre-commit

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx --no-install lint-staged
```

Comment:
Did we decide if this was the proper way to run this?

We're not using the --no-install flag [in pulse](https://github.com/artsy/pulse/blob/adae00989a605c5b0b99006b9f16107270b51605/.husky/pre-commit#L4).

I think it makes sense to include this flag, but we might also want to add it to pulse.

Reply:
I was just following [their examples](https://typicode.github.io/husky/#/?id=locally-installed-binaries) which all seem to do this. Reading the related discussions (like https://github.com/zkat/npx/issues/198), it seems like the recommended way to do this for already-installed dependencies, as we wouldn't want this hook to modify `package.json` or `yarn.lock`.

In our mob, we tried removing it while debugging, but the problems ended up being unrelated. I.e., I think we probably _should_ add a similar flag to Pulse and elsewhere.